### PR TITLE
setup label sync

### PR DESCRIPTION
**Pull request by fryorcraken**

### CODE REVIEW

File: .github/labels.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  color: 06B6C8
+- name: track:sdks
+  description: SDKS track (Waku Product), including bindings
+  color: 34D557
+- name: track:go-waku-productionization
+  description: go-waku productionization track (Waku Product)
+  color: 9DEA79
+
+# Epics/Milestones
+- name: E:2023-light-protocols
```
Comment (fryorcraken) [inline .github/labels.yml:90]: New PPG tool will enable grouping issues via epic label, prefixed with `E:`. Starting to set this up.

File: .github/labels.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  description: Infra, devops, CI and related tasks
+  color: 277196
+- name: milestone
+  description: Milestone issue with a subset of issues within a specific track
+  color: 1CC0B0
+- name: test
+  description: Issue related to the test suite with no expected consequence to production code
+  color: 277196
+
+# Tracks
```
Comment (fryorcraken) [inline .github/labels.yml:45]: @jm-clius are all these track still relevant for you? We can discuss the epic label usage.

File: .github/workflows/sync-labels.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+jobs:
+  sync-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: micnncim/action-label-syncer@v1
+        with:
+          manifest: .github/labels.yml
+          repository: |
+            waku-org/docs.waku.org
```
Comment (fryorcraken) [inline .github/workflows/sync-labels.yml:18]: Did not enter ALL repos as some are not that relevant in terms of tracking work but anyone can add a repo.

**Review by jm-clius — APPROVED** [review]
Generally LGTM. Assuming a next step would be to define the actual epics?

**Comment by fryorcraken** [comment]
> Generally LGTM. Assuming a next step would be to define the actual epics?

Yes so my latest proposal was:

The "Epics" described under https://notes.status.im/Uz9HeCwZTDSYyOq36Q54cA (e..g the one in monthly report) get a label issue and an issue in this repo.

Then, we can create milestones on each relevant repo (nwaku, js-waku, etc). Each milestone related to an Epic.
In our new weekly update, we report update on active milestones.
milestones get an Epic label.

Then, milestones can have several smaller issues related.
This issues should be "checked boxed" in the milestone and also get the Epic label.

I hope it will make sense :)

**Pull request merged**